src/runtime.js

```diff
@@ -861,9 +861,6 @@ class MultidocManager {
  * @return {boolean}
  */
 function maybeLoadCorrectVersion(win, fnOrStruct) {
-  if (!isExperimentOn(win, 'version-locking')) {
```

Comment:
Noo, I actually use this in development all the time! I frequently rebuild the runtime without extensions, and without this it will prevent any extensions from being loaded.

Reply:
@jridgewell Are you saying you want to rebuild with `gulp --noextensions`, and then load the extensions that were built by a prior call to `gulp build`?

Reply:
If so, can this functionality not be moved to a local development flag? Does it have to be an experiment?

Reply:
Im perfectly ok leaving this if you find it useful

Reply:
Development flag would be fine.

With the new `--noextensions` and `--extension=<foo>` flags, it's likely people will run into this much more. Before, it's likely that I'm the only person who was building runtime without extensions (I commented them out in gulpfile.js).